### ISSUE TITLE
return value should include the css function

### ISSUE THREAD

**Issue by xiweixiong** [v-vibe/vue-styled-components#13]
### Is your feature request related to a problem? Please describe

`${(props) => {}}`

The return value is only string or number
I can't write complex logic or return complex css

I hope to achieve
`${(props) => expression ? css`` : css``}`

**Comment by akinoccc** [comment]
Could you provide a more specific example?

**Comment by akinoccc** [comment]
Do you want like this?

```vue
<script setup>
const testEmbedCss1 = css`
  background: white;
`
const testEmbedCss2 = css`
  background: blue;
`
const TestEmbedComponent = styled('div', { show: Boolean })`
  ${(props) => {
    return props.show ? testEmbedCss1 : testEmbedCss2
  }}
`
</script>
```

**Comment by xiweixiong** [comment]
> @xiweixiong你想要這樣嗎？
> 
> ```
> <script setup>
> const testEmbedCss1 = css`
>   background: white;
> `
> const testEmbedCss2 = css`
>   background: blue;
> `
> const TestEmbedComponent = styled('div', { show: Boolean })`
>   ${(props) => {
>     return props.show ? testEmbedCss1 : testEmbedCss2
>   }}
> `
> </script>
> ```

yes

**Comment by akinoccc** [comment]
It should be supported, but now I've found a bug that causes it to not run properly. Once I fix it, the above code will be supported. Thank you for opening the issue!

**Comment by akinoccc** [comment]
@xiweixiong I released a beta version [v1.3.0-beta.1](https://github.com/v-vibe/vue-styled-components/releases/tag/v1.3.0-beta.1) and you can try it.
